server/http/routes/api/polls.js

```diff
@@ -0,0 +1,92 @@
+const auth = require('../middlewares/auth')
+
+module.exports = (plasma, dna, helpers) => {
+  return {
+
+    'GET': [
+      auth.authorize(dna.jwtSecret),
+      (req, res, next) => {
+        const listChemical = {
+          type: 'polls-list'
+        }
+        plasma.emit(listChemical, (err, polls) => {
+          if (err) return next(err)
+          res
+            .status(200)
+            .send(polls)
+
+          return next()
+        })
+      }
+    ],
+
+    'POST': [
+      auth.authorize(dna.jwtSecret),
+      (req, res, next) => {
+        // TODO: validate
+        const createChemical = {
+          type: 'polls-create',
+          args: {
+            userId: req.user._id,
+            rate: req.body.rate
+          }
+        }
+        plasma.emit(createChemical, (err, poll) => {
+          if (err) return next(err)
+          res
+            .status(200)
+            .send(poll)
+
+          return next()
+        })
+      }
+    ],
+
+    'PUT /:id': [
+      auth.authorize(dna.jwtSecret),
+      (req, res, next) => {
+        // TODO: validate
+        // user should not be able to add votes to himself
+        // if approved status changes, put completedAt value as well
```

Comment:
1) we can discuss where to put validations. I guess we can validate the `args` in the route before passing them to the organelle.
2) you've already completed :)